src/main/webapp/app/shared/util/firebase/firebase-utils.tsx

```diff
@@ -400,8 +400,12 @@ const addDuplicateMutationInfo = (duplicates: DuplicateMutationInfo[], mutationN
   }
 };
 
+export const hasMultipleMutations = (mutationName: string) => {
+  return mutationName.includes(',');
+};
 export const isMutationEffectCuratable = (mutationName: string) => {
-  if (mutationName.includes(',')) {
+  const multipleMuts = hasMultipleMutations(mutationName);
+  if (multipleMuts) {
```

Comment:
This is probably not in the scope of this PR, but we should update so that `grch37:L265P, grch38:L252P` should not get matched.